src/redprl/syntax/variable_kit.sml

```diff
@@ -23,4 +23,6 @@ struct
 
   val substMany = RedPrlAbt.substVarenv o ctxFromList
   fun subst s = substMany [s]
+
+  fun alphaRenameTubes w = List.map (fn (eq, (u, tube)) => (eq, rename (w, u) tube))  
```

Comment:
@jonsterling Not sure if this function belongs to the variable kit...

Reply:
It's ok, we can find a better place for it later.